prowler/providers/aws/services/ecr/ecr_service.py

```diff
@@ -47,6 +47,8 @@ def _describe_registries_and_repositories(self, regional_client):
                                 scan_on_push=repository["imageScanningConfiguration"][
                                     "scanOnPush"
                                 ],
+                                immutability=repository["imageTagMutability"]
+                                == "IMMUTABLE",
```

Comment:
```suggestion
                                immutability=repository.get("imageTagMutability", "MUTABLE")
```

Please, compare the strings in the check.